docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -58,37 +86,23 @@ Now, let's tell the `native-image` executable about this configuration file by t
         <id>native</id>
         <properties>
             <quarkus.package.type>native</quarkus.package.type>
-            <quarkus.native.additional-build-args>-H:ResourceConfigurationFiles=${project.basedir}/resources-config.json</quarkus.native.additional-build-args>
+            <quarkus.native.additional-build-args>-H:ResourceConfigurationFiles=resources-config.json</quarkus.native.additional-build-args>
         </properties>
     </profile>
 </profiles>
 ----
 
-[TIP]
-====
-Multiple options may be separated by a comma.
-
-Another possibility is to include the `quarkus.native.additional-build-args` configuration property in your `application.properties`.
-====
-
-When using Gradle, we need to configure the additional build arguments in the plugin configuration:
+When using Gradle, configuring the `buildNative` task like so suffices:
 
 [source,groovy]
 ----
 buildNative {
     additionalBuildArgs = [
-            "-H:ResourceConfigurationFiles=${buildDir}/resources/main/resources-config.json".toString()
+            'resources-config.json'
```

Comment:
Don't you need to still specify the `-H:ResourceConfigurationFiles` here?

Reply:
Oops!

Reply:
Fixed